.github/workflows/kind-e2e.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
 
         test-suite:
           - ./test/e2e
+          - ./test/e2e_new
```

Comment:
Can we just use `./test/...` or something like that? Fewer actions consumed from the shared poll.

Reply:
I'm worried there might be problems because it's going to build a single test binary...

Reply:
Ok, no problem!